lib/bindConnectors/bindTriggerRequest.js

```diff
@@ -2,6 +2,13 @@ var _                = require('lodash');
 var when             = require('when');
 var parseRequestBody = require('./parseRequestBody');
 
+function formatReject (body) {
+	//Catch will resolve the promise, ergo explicitly reject again
+	return when.reject({
+		headers: {},
```

Comment:
Why doesn't the reject have a version but the resolve does?

Reply:
I don't think BE has a separate error handler for `version: 2` - well, BE has not requested it so far.